.eslintrc.yaml

```diff
@@ -14,4 +14,6 @@ globals:
   exports: true
   uneval: false
   crypto: true
+  document: true
+  self: true
```

Comment:
These should go in `data/.eslintrc`, since they only apply to the code written there.